yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -2223,7 +2223,7 @@ func (yb *TargetYugabyteDB) NumOfLogicalReplicationSlots() (int64, error) {
 	return numOfSlots, nil
 }
 
-func (yb *TargetYugabyteDB) GetTablesHavingExpressionUniqueIndexes(tableNames []sqlname.NameTuple, considerPartitions bool) ([]sqlname.NameTuple, error) {
+func (yb *TargetYugabyteDB) GetTablesHavingExpressionUniqueIndexes(tableNames []sqlname.NameTuple, returnPartitionRootTable bool) ([]sqlname.NameTuple, error) {
```

Comment:
pls add documentation as a comment for what `returnPartitionRootTable` does. It's quite complicated because it seems to depend on the input `tableNames` as well.